cypress/e2e/ModelizerPage/TextView/git/filter.branch.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Test modelizer text view: filter git branch
+
+  Background:
+    Given I clear cache
+    And I set context field "repository_url" with "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set context field "projectName" with "leto-modelizer-project-test"
+    And I visit the "/"
+
+    When I click on "[data-cy=\"import-project\"]"
+    And I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-repository-input\"]" text "{{ repository_url }}"
+    And I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And I click on "[data-cy=\"import-project-form\"] [data-cy=\"import-project-form-submit\"]"
+    Then I expect "positive" toast to appear with text "Project has been imported 🥳!"
+    And I expect "[data-cy=\"import-project-form\"]" is closed
+    And I visit the "/#/modelizer/{{projectName}}/text"
+
+  Scenario: Expect to have no branch with "no_main" filter
+    When I click on "[data-cy=\"git-current-branch\"]"
+    Then I expect "[data-cy=\"git-menu-branch-local-main\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-remote-main\"]" exists
+
+    When I set on "[data-cy=\"git-branch-menu-search-bar\"]" text "not_main"
+    Then I expect "[data-cy=\"git-menu-branch-local-main\"]" not exists
+    And I expect "[data-cy=\"git-menu-branch-remote-main\"]" not exists
+
+  Scenario: Expect to have main branch with "main" filter
+    When I click on "[data-cy=\"git-current-branch\"]"
+    Then I expect "[data-cy=\"git-menu-branch-local-main\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-remote-main\"]" exists
+
+    When I set on "[data-cy=\"git-branch-menu-search-bar\"]" text "main"
+    Then I expect "[data-cy=\"git-menu-branch-local-main\"]" exists
+    And I expect "[data-cy=\"git-menu-branch-remote-main\"]" exists
+
+  Scenario: Expect to have main branch with "no_main main" filter
```

Comment:
Hard to know if the test works if result does not change.

Reply:
yeah but the test work !

Reply:
nope

Reply:
the test is change done